paasta_tools/chronos_tools.py

```diff
@@ -257,6 +257,13 @@ def get_disabled(self):
     def get_schedule(self):
         return self.config_dict.get('schedule')
 
+    def get_schedule_interval_in_seconds(self):
+        try:
+            _, _, interval = self.get_schedule().split('/')
```

Comment:
I'd really like us to be able to do this for crontab too - can you find a library that'll let us calculate the next and last run, and use the delta as interval?